src/app/shared/models/raid.js

```diff
@@ -6,7 +6,11 @@ export const raidSchema = new Schema({
     required: true,
   },
   description: String,
-  location: [Number],
+  location: {
+    type: [Number],
+    required: true,
+    index: '2d'
```

Comment:
Oh yeah, sorry I took that out. I was a bit confused by it. I notice the Mongo docs indicates to use this if:

- your database has legacy location data from MongoDB 2.2 or earlier, and
- you do not intend to store any location data as GeoJSON objects.

I looked up [GeoJSON](http://geojson.org/geojson-spec.html), and it may be something we would like to use. There have been discussions of possibly creating notifications for when a user leaves 'Sanctuary' cities. This is *far* down the line, but what do you think of using GeoJSON right off the bat?